backend/app/api/v1/integrations.py

```
@@ -9,101 +9,156 @@
+)
+from app.core.logging_config import get_logger, log_performance
 
 router = APIRouter()
+logger = get_logger(__name__)
 
 
 @router.post("/", response_model=IntegrationResponse, status_code=status.HTTP_201_CREATED)
+@log_performance("create_integration")
 async def create_integration(
     request: IntegrationCreateRequest,
     user_id: UUID = Depends(get_current_user)
```

Ruff (0.14.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)